chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -27,9 +25,14 @@
 
 tftproot = node[:provisioner][:root]
 
-pxecfg_dir="#{tftproot}/discovery/pxelinux.cfg"
-pxecfg_default="#{tftproot}/discovery/pxelinux.cfg/default"
-uefi_dir="#{tftproot}/discovery"
+# This is what we could support, but this requires validation
+#discovery_arches = ['x86_64', 'ia32']
+discovery_arches = ['x86_64']
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)